docs/specs/src/hwp_1_intro.md

```diff
@@ -0,0 +1,87 @@
+---
+title: 'Holochain'
+subtitle: 'Distributed Coordination by Scaled Consent not Global Consensus'
+author:
+ - Eric Harris-Braun
+ - Arthur Brock
+abstract: |
+ In this paper we present a frame and a specification for a generalized system for
+ large scale distributed coordination that does not rely on global consensus. We start
+ with some axioms; we proceed to describe the consequences of these axioms, looking
+ at the problem through the lens of scaling consent; we present an informal description
+ of the system; we provide a more formal outline of the necessary integrity guarantees
+ and system components needed to deliver those guarantees; and finally we conclude with
+ a high-level yet sufficiently detailed technical specification of our implementation
+ of such a system.
+documentclass: 'revtex4-1'
+---
+
+# Introduction
+
+## Preamble
+
+The alpha version of the [Holochain white paper](https://github.com/holochain/holochain-proto/blob/whitepaper/holochain.pdf) took a very formal approach to describing generalized distributed computation schemes and carefully compared the approach taken by Blockchain-based systems with the approach taken by Holochain as well as providing formal reasoning for the benefits of Holochain's agent-centric approach.
+
+In this second version, we describe Holochain on its own terms by first providing the highest level view of our approach and our starting axioms. Then, at the next level of detail we provide an informal system description using the metaphor of games. Then we dive into deeper detail with a formal system description including the specific context and assumptions we come from, the integrity guarantees Holochain offers, and a formal state model and discussion of security and safety concerns. Finally, at the most detailed level, we provide an implementation specification in Appendix A.
+
+## Axioms
+
+We begin by stating our axioms[^byzantine] regarding the nature of coordination:
+
+[^byzantine]: Some readers may come to the problems of distributed coordination from the framework laid out by the literature on Byzantine Fault Tolerance. These axioms and a discussion of why to start with them are explained in our paper [[The Players of Ludos: Beyond Byzantium](https://docs.google.com/document/d/1HBNgIooElD5widCuX9XmiOzbVIpEF5XXH67mZbnUFjo/edit#).
```

Comment:
```suggestion
[^byzantine]: Some readers may come to the problems of distributed coordination from the framework laid out by the literature on Byzantine Fault Tolerance. These axioms and a discussion of why to start with them are explained in our paper [The Players of Ludos: Beyond Byzantium](https://docs.google.com/document/d/1HBNgIooElD5widCuX9XmiOzbVIpEF5XXH67mZbnUFjo/edit#).
```

Also wouldn't a permalink be better here, or we can be guaranteed that this link will always be valid?

Reply:
It'd be much better. @zippy can we put this somewhere?